pkg/controller/garbagecollector/garbagecollector.go

```diff
@@ -519,7 +521,8 @@ func (gc *GarbageCollector) attemptToDeleteItem(item *node) error {
 			// otherwise, default to background.
 			policy = metav1.DeletePropagationBackground
 		}
-		klog.V(2).Infof("delete object %s with propagation policy %s", item.identity, policy)
+		klog.V(2).InfoS("Deleting object", strings.ToLower(item.identity.Kind), klog.KRef(item.identity.Namespace, item.identity.Name),
+			fmt.Sprintf("%s_uid", strings.ToLower(item.identity.Kind)), item.identity.UID, "propagation_policy", policy)
```

Comment:
```suggestion
			fmt.Sprintf("%s_uid", strings.ToLower(item.identity.Kind)), item.identity.UID, )
				klog.V(2).InfoS("Deleting object", "object", klog.KObj(item.identity), "kind":  item.identity.Kind, "group", item.identity.Group, "propagation_policy", policy)
```

Reply:
`item.identity` doesn't have group, skipping
updated `propagation_policy` to `propagationPolicy`

Reply:
ack